docs/sdk-guide.md

```diff
@@ -816,13 +816,15 @@ Find example code at [CarbonReaderExample](https://github.com/apache/carbondata/
    * This method return the version details in formatted string by reading from carbondata file
    * If application name is SDK_1.0.0 and this has written the carbondata file in carbondata 1.6 project version,
    * then this API returns the String "SDK_1.0.0 in version: 1.6.0-SNAPSHOT"
-   * @param dataFilePath complete path including carbondata file name
+   * Default value of validate is false, it won't validate the version details between different carbondata files.
```

Comment:
" Default value of validate is false,"
- This statement is not valid as we don't have validate and no validate flows differently.

Reply:
ok, removed